scripts/mcp/proxysql_mcp_stdio_bridge.py

```
@@ -0,0 +1,362 @@
+import asyncio
+import json
+import os
+import sys
+from typing import Any, Dict, Optional
+from datetime import datetime
+
+import httpx
+
+# Minimal logging to file for debugging
+# Log path can be configured via PROXYSQL_MCP_BRIDGE_LOG environment variable
+_log_file_path = os.getenv("PROXYSQL_MCP_BRIDGE_LOG", "/tmp/proxysql_mcp_bridge.log")
```

Ruff (0.14.11)

Probable insecure usage of temporary file or directory: "/tmp/proxysql_mcp_bridge.log"

(S108)